lms/static/scripts/frontend_apps/components/ContentSelector.tsx

```diff
@@ -188,6 +193,15 @@ export default function ContentSelector({
   const selectCanvasPage = (page: File | Page) =>
     selectPageAsURL(page, 'Canvas');
 
+  const selectCanvasStudio = (video: File | Page) => {
+    cancelDialog();
+    onSelectContent({
+      type: 'url',
+      url: video.id,
+      name: `Canvas Studio video: ${video.display_name}`,
+    });
+  };
```

Comment:
This can be slightly simplified now

```suggestion
  const selectCanvasStudio = (video: File | Page) => {
    const name = `Canvas Studio video: ${video.display_name}`;
    selectURL(video.id, name);
  };
```

But [your comment here](https://github.com/hypothesis/lms/pull/6104/files#r1517431273) is still correct and we will be able to simplify it even further.

Reply:
OK, I'll do this. `name` is probably not the best parameter name here, as the text is more of a "description".